code/modules/paperwork/handlabeler.dm

```diff
@@ -116,6 +116,15 @@
 	labels_left = initial(labels_left) //Yes, it's capped at its initial value
 	return ITEM_INTERACT_SUCCESS
 
+/obj/item/hand_labeler/examine()
+	. = ..()
+	if(labels_left > 1)
+		. += span_notice("It looks like it could label [labels_left] more things.")
+	if(labels_left == 1)
+		. += span_notice("It looks like it could label [labels_left] more thing.")
+	if(labels_left == 0)
+		. += span_notice("It's out of labels.")
```

Comment:
```suggestion
	switch(labels_left)
		if(2 to INFINITY)
			. += span_notice("It looks like it could label [labels_left] more things.")
		if(1)
			. += span_notice("It looks like it could label [labels_left] more thing.")
		if(0)
			. += span_warning("It's out of labels.")
```